src/components/ReportTransaction.js

```diff
@@ -16,21 +18,72 @@ const propTypes = {
 
     /** The report action which we are displaying */
     action: PropTypes.shape(ReportActionPropTypes).isRequired,
+
+    /** Can this transaction be rejected? */
+    isRejectable: PropTypes.bool,
+
+    /** Is the authenticated user the creator of this transaction? */
+    isCurrentUserTransactionCreator: PropTypes.bool,
```

Comment:
This prop is only controlling the button text, so I think it would be better to name this `buttonLabelText` and passing the appropriate translate key, or possibly changing this to be `showDeclineLabel`. My thinking is that this component doesn't care whether or not the user was the transaction creator, it only cares which label to show for the button.

Reply:
That makes sense to me, will improve the prop name.